### PR TITLE
[WFCORE-1591] add missing help message for --properties argument.

### PR DESCRIPTION
**Pull request by soul2zimate**
https://issues.jboss.org/browse/WFCORE-1591

### CODE REVIEW

File: cli/src/main/resources/help/help.txt [hunk 1]
```diff
@@ -73,6 +73,10 @@ Usage:
                    will be prompted to enter the password when the connect
                    command is issued.
 
+ --properties    - specifies a path to a properties file which contains
+                   property value pairs to define system property.
```
Comment (bstansberry) [inline cli/src/main/resources/help/help.txt]: s/system property/system properties/g

File: cli/src/main/resources/help/help.txt [hunk 2]
```diff
@@ -73,6 +73,10 @@ Usage:
...
+                   the properties file uses a standard KEY=VALUE syntax.
```
Comment (bstansberry) [inline cli/src/main/resources/help/help.txt]: s/the/The

**Comment by bstansberry** [comment]
@aloubyansky please approve

File: cli/src/main/resources/help/help.txt (hunk 1 above)
Comment (aloubyansky) [inline cli/src/main/resources/help/help.txt]: Please, add it also to the list of alternatives below Usage. Thanks!

**Comment by soul2zimate** [comment]
Thanks, it's been rebased.

**Comment by aloubyansky** [comment]
Looks good to me. Thanks!

**Comment by bstansberry** [comment]
retest this please

Failure is unrelated

**Pull request merged**